app/views/teacher_interface/registration_number/edit.html.erb

```diff
@@ -6,8 +6,12 @@
 
   <%= f.govuk_text_field :registration_number, label: { size: "l", tag: "h1" } %>
 
-  <% if @teaching_authority_other.present? %>
-    <%= raw GovukMarkdown.render(@teaching_authority_other) %>
+  <% if @application_form.region.status_check_online? %>
+    <%= raw GovukMarkdown.render(@application_form.region.teaching_authority_status_information) %>
+  <% end %>
+
+  <% if @application_form.region.sanction_check_online? %>
+    <%= raw GovukMarkdown.render(@application_form.region.teaching_authority_sanction_information) %>
   <% end %>
```

Comment:
We also need to show this content on the eligible page, again depending on whether it's a written/online country, like on this slide: https://docs.google.com/presentation/d/1gcUgk4Lm4amlCNdSQVNEYqy85YeeD6JUwJ1c85U8VyM/edit#slide=id.g198a5499b8b_0_187